diesel_cli/src/cli.rs

```diff
@@ -108,21 +108,39 @@ pub fn build_cli() -> App<'static, 'static> {
                 .index(1)
                 .takes_value(true)
                 .multiple(true)
-                .help("Table names to filter (default whitelist if not empty)"),
+                .help("Table names to filter (default include if not empty)"),
```

Comment:
Should this say `only-tables`? TBH I wonder if we should just remove any "default" behavior (I think we have an issue open saying that the default is broken anyway)

Reply:
If we remove the default behavior would it make more sense to have `--only-tables` and `--except-tables` take a comma separated list of arguments instead of relying on the positional arguments to the command? (This was confusing when I first encountered it, from glancing at the help I really expected `--blacklist=foo,bar,baz` instead of `--blacklist foo bar baz`)